include/behaviortree_ros/register_ros_plugin.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 Davide Faconti
```

Comment:
Is this the right copyright?

Reply:
this file was adapted from https://github.com/BehaviorTree/BehaviorTree.ROS2/blob/humble/include/behaviortree_ros2/plugins.hpp in the BT.ROS2 repo, the copyright is from there

Reply:
ok, so since we adapted it, we can also add us to the copyright